docs/guides/networking/linode-network/ip-sharing-atlanta/index.md

```diff
@@ -0,0 +1,265 @@
+
+{{< note >}}
+This guide discusses a temporary workaround specific to the Atlanta data center. Future planned network upgrades may impact this feature again and may require additional configuration.
+{{</ note >}}
+
+## Before You Begin
+
+1. Prior to beginning the process outlined in this guide, make sure that you have received an IPv4 address(es) from Linode Support to use as your Elastic IP(s). To request an additional IPv4 address, [open a new support ticket from the Cloud Manager](/docs/guides/support/#contacting-linode-support).
+
+1. Ensure you have set the [hostname](/docs/getting-started/#set-the-hostname) and have updated the [hosts file](/docs/getting-started/#update-your-system-s-hosts-file) on your Compute Instances.
+
+1. Ensure Python 3 is installed on your system. See [FRR's official documentation](http://docs.frrouting.org/en/latest/installation.html#python-dependency-documentation-and-tests) to learn about FRR's Python dependencies.
+
+1. [Disable Network Helper](/docs/platform/network-helper/#single-per-linode) on the Elastic IP Linodes and reboot them.
+
+## Install FRR
+
+This section provides instructions for installing FRR on Debian, Ubuntu, and CentOS systems through their native package managers. If you're using a different distribution or prefer to install FRR from source, follow [FRR's official installation instructions](http://docs.frrouting.org/en/latest/installation.html) to install FRR using git.
+
+### Ubuntu and Debian
+
+**Supported distributions:** *Ubuntu 20.04, 18.04, and 16.04 | Debian 11, 10, and 9*
+
+1.  Install the package dependencies needed to securely install FRR on your system:
+
+        sudo apt-get install apt-transport-https gnupg
+
+1.  Add FRR's GPG key:
+
+        curl -s https://deb.frrouting.org/frr/keys.asc | sudo apt-key add -
+
+1.  Set the FRR environment variable as `frr-stable` to install the latest FRR release. Specific releases can also be targeted by setting the variable to `frr-8`, `frr-7`, `frr-6` (see [FRR Debian Repository](https://deb.frrouting.org/) for more details).
+
+        FRRVER="frr-stable"
+
+1.  Add FRR's Debian repository to your system's source's list:
+
+        echo deb https://deb.frrouting.org/frr $(lsb_release -s -c) $FRRVER | sudo tee -a /etc/apt/sources.list.d/frr.list
+
+1.  Install the FRR package:
+
+        sudo apt update && sudo apt install frr frr-pythontools
+
+### CentOS/RHEL
+
+**Supported distributions:** *CentOS Stream (and 8), CentOS 7, other RHEL derivatives (including AlmaLinux 8, and Rocky Linux 8), and Fedora.*
```

Comment:
@wildmanonline not all of these support distributions are given instructions in the Configure FRR section (CentoS Stream, 7, and all the RHEL derivatives + Fedora). Should we consider pruning the list of supported distributions here? Or tell the reader that configuration instructions for those distros are outside the scope of the guide?

Reply:
I updated the distros under the Configure FRR section to include all supported distros.